lib/mix/tasks/ash_oban.install.ex

```diff
@@ -0,0 +1,120 @@
+defmodule Mix.Tasks.AshOban.Install.Docs do
+  @moduledoc false
+
+  def short_doc do
+    "Installs AshOban and Oban"
+  end
+
+  def example do
+    "mix igniter.install ash_oban"
+  end
+
+  def long_doc do
+    """
+    #{short_doc()}
+
+    ## Example
+
+    ```bash
+    #{example()}
+    ```
+    """
+  end
+end
+
+if Code.ensure_loaded?(Igniter) do
+  defmodule Mix.Tasks.AshOban.Install do
+    @shortdoc "#{__MODULE__.Docs.short_doc()}"
+
+    @moduledoc __MODULE__.Docs.long_doc()
+
+    use Igniter.Mix.Task
+
+    @impl Igniter.Mix.Task
+    def info(_argv, _composing_task) do
+      %Igniter.Mix.Task.Info{
+        # Groups allow for overlapping arguments for tasks by the same author
+        # See the generators guide for more.
+        group: :ash,
+        # dependencies to add
+        adds_deps: [],
+        # dependencies to add and call their associated installers, if they exist
+        installs: [
+          {:oban, "~> 2.0"}
```

Comment:
This version of Oban was released in July 2020. It's far older than we'd recommend anybody use, and not fully compatible with newer dependencies. The `ash_oban` package itself is only a couple of years old. Perhaps specify a more recent minimum version, maybe `~> 2.17`?

Of course, the only minor release that has an ash installer is v2.19, so perhaps it doesn't matter?

Reply:
It shouldn't really matter, but the real reason I did that is because `mix igniter.install` pins to the major version only, and if I put a higher version here the user will be prompted to change it to that higher version, even they they are certainly on a compatible version. 

When oban task itself was updating the version, if ash_oban had a different dep string, then lyou could be asked to confirm the version 3 times consecutively 😂. Once for oban install, once for oban setting it, once for ash_oban setting it to the higher value.

But it's an interesting point that theoretically this would allow older versions. It could only happen if a dep was pinning oban to something older though, which seems unlikely.